internal/cli/client_ratelimit_test.go

```diff
@@ -0,0 +1,175 @@
+package cli
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func skipIfDisabled(t *testing.T) {
+	if os.Getenv("PORT_RATE_LIMIT_DISABLED") != "" {
+		t.Skip("Skipping rate limit test because PORT_RATE_LIMIT_DISABLED is set")
+	}
+}
```

Comment:
Add `t.Helper()` to helper functions. It makes the test runner ignore these functions in stack traces.

```suggestion
func skipIfDisabled(t *testing.T) {
	t.Helper()
	if os.Getenv("PORT_RATE_LIMIT_DISABLED") != "" {
		t.Skip("Skipping rate limit test because PORT_RATE_LIMIT_DISABLED is set")
	}
}
```